src/http/cookie.cr

```diff
@@ -152,7 +152,12 @@ module HTTP
 
       def parse_cookies(header)
         header.scan(CookieString).each do |pair|
-          yield Cookie.new(pair["name"], pair["value"])
+          value = pair["value"]
+          if value.starts_with?('"')
```

Comment:
may be check end_with also?

Reply:
That's net necessary because the regex makes sure there are delimiters on both ends.